OutOfSchool/OutOfSchool.WebApi/Controllers/V1/TagController.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System.Net.Mime;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Localization;
+using OutOfSchool.BusinessLogic.Enums;
+using OutOfSchool.BusinessLogic.Models.Tag;
+
+namespace OutOfSchool.WebApi.Controllers.V1;
+
+[ApiController]
+[ApiVersion("1.0")]
+[Route("api/v{version:apiVersion}/[controller]/[action]")]
+public class TagController : ControllerBase
+{
+    private readonly ITagService service;
+    private readonly IStringLocalizer<SharedResource> localizer;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="TagController"/> class.
+    /// </summary>
+    /// <param name="service">Service for Tag model.</param>
+    /// <param name="localizer">Localizer.</param>
+    public TagController(ITagService service, IStringLocalizer<SharedResource> localizer)
```

Comment:
(I don't insist) I find very it simple and useful to use primary constructors (without related explicit fields - using implicit fields from generated by C# constructor parameter) when constructor contains only services injection.

There are some drawbacks while using autogenerated (implicit) fields
- when explicit fields aren't used the autogenerated ones are not readonly;
- also you can't put any null-check or other validation logic for that parameters.

But code is shorter and it's pretty rare case when previous drawbacks really become an issues.

Reply:
Please write a note here clarifiing your plans to perform any changes or not. 
At the moment I see this comment open, heving no response and no changes of related code (primary constructor) implemented

Reply:
I have followed a path which similar to the one which has already been used in the project (for instance SocialGroupController), therefore there is no need for changing this (at least I do not find it, maybe there is a big reason to do it).